cddl/contrib/opensolaris/lib/libdtrace/common/dt_link.c

```diff
@@ -425,7 +427,11 @@ prepare_elf64(dtrace_hdl_t *dtp, const dof_hdr_t *dof, dof_elf64_t *dep)
 
 		for (j = 0; j < nrel; j++) {
 #if defined(__aarch64__)
-/* XXX */
+#define R_AARCH64_ABS64			257
+		rel->r_offset = s->dofs_offset +
```

Comment:
This line needs an extra tab at the start as well.

Reply:
thanks